src/__tests__/__snapshots__/Badge.js.snap

```diff
@@ -6,7 +6,6 @@ exports[`Badge bg blue sets background-color and color 1`] = `
   display: inline-block;
   font-size: 12px;
   font-weight: 600;
-  -webkit-text-transform: uppercase;
```

Comment:
Not sure why snapshots would've updated, unless you ran `npm test` – let's sync up to make sure your local setup doesn't do this when editing docs